### DISCUSSION TITLE
기획 문서에 관한 질문들 (상점 관련)

### DISCUSSION THREAD

**Discussion by mille-feu** [Tiny-Metro/Poly-Metro#57] [Q&A]
- 업그레이드의 적용 방식에 관한 부가적인 설명이 필요합니다.
  - 여러 번 업그레이드가 가능한지
  - 업그레이드는 각각의 동차, 객차, 역에 따로 적용되는지
  - 따로 적용될 경우, 구매한 업그레이드 횟수의 사용은 어떻게 하는지
- 기획 문서의 "동차 구매와 동차 업그레이드는 따로"가 의미하는 바에 대한 설명이 필요합니다
- 역 업그레이드 시 증가하는 수용 인구는 게임 시스템적으로 어떤 의미가 있는지 설명이 필요합니다.

**Comment by cstrnull00** [comment]
* 업그레이드 적용 방식
  * 업그레이드는 각 개체별로 한 번씩만 가능
  * 각각 동차, 객차, 역에 따로 적용
  * 질문에 대해 더 설명이 필요합니다.
* 동차를 구매 -> 새로운 동차가 노선에 독립적으로 추가
* 동차를 업그레이드 -> 노선에 추가되어있는 동차의 승객 수용량이 늘어남
* 역에서 대기 승객이 6명 이상일 때부터 불만도가 증가하는데, 이 수용 인구를 늘려주는 역할을 합니다.

**Reply by cstrnull00** [reply]
[이벤트](https://github.com/Tiny-Metro/Tiny-Metro/wiki/Document:-%EC%9D%B4%EB%B2%A4%ED%8A%B8) 참조
* 동차, 객차 업그레이드에 대한 내용 변경
